src/client/runner/automation/playback/click.js

```diff
@@ -249,13 +249,15 @@ export default function (el, options, runCallback, errorCallback) {
         mousedown: function (callback) {
             //NOTE: in webkit and ie raising mousedown event opens select element's dropdown,
             // therefore we should handle it and hide the dropdown (B236416)
-            var needHandleMousedown = (browserUtils.isWebKit || browserUtils.isIE) && domUtils.isSelectElement(el),
-                wasPrevented        = null,
-                blurRaised          = false,
-                activeElement       = domUtils.getActiveElement(),
+            var needHandleMousedown   = (browserUtils.isWebKit || browserUtils.isIE) && domUtils.isSelectElement(el),
```

Comment:
this is a part of the task in the `Before APIv2` milestone (https://github.com/superroma/testcafe-phoenix/issues/50)